op-challenger/fault/alphabet_provider_test.go

```diff
@@ -8,6 +8,58 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// TestAlphabetProvider_Positions tests the [fault.AlphabetProvider] interfaces correctly
+// with the [fault.Position] type.
+func TestAlphabetProvider_Positions(t *testing.T) {
```

Comment:
Unless you use `NextMove` you're only really testing the `Position` object here.
The expectedHash check is really testing the AlphabetProvider.